tests/modules/cluster.c

```diff
@@ -34,13 +34,52 @@ int test_cluster_shards(ValkeyModuleCtx *ctx, ValkeyModuleString **argv, int arg
     return VALKEYMODULE_OK;
 }
 
+#define MSGTYPE_PING 1
+#define MSGTYPE_PONG 2
+
+/* test.pingall */
+int PingallCommand_ValkeyCommand(ValkeyModuleCtx *ctx, ValkeyModuleString **argv, int argc) {
```

Comment:
This function doesn't follow the naming style in this module. The others are snake case and called `test_cluster_slots` and similar. How about `pingall_command`?

Is there any risk of confusion with this module's PING/PONG and the cluster bus PING/PONG? Maybe call these something else? DING/DONG?

Reply:
It's just copy/paste from `hellocluster.c`. I didn't care much. Let me do a bit of cleanup.